packages/quantic/force-app/main/default/lwc/quanticTimeframeFacet/quanticTimeframeFacet.js

```diff
@@ -322,7 +322,7 @@ export default class QuanticTimeframeFacet extends LightningElement {
   get hasActiveValues() {
     return (
       this.formattedValues.some((v) => v.selected) ||
-      this.dateFilterState?.range
+      !!this.dateFilterState?.range
```

Comment:
You're going to have to explain this one lol

Reply:
That's nothing really, just a better way to see the return value of this getter, and this is not the fix fo the issue.

Before this change, `hasActiveValues` is truthy when `this.dateFilterState?.range` is  truthy, but since this latter is not a boolean, it means that the return value  of this getter will not be a boolean, it can be truthy or falsy but not a boolean, in other words not a simple value `true` or `false`. 
Which is something that can be a bit misleading while debugging the code.

So instead, now I'm making the return value of this getter a proper boolean, it can only be `true` or `false` no truthy/falsy  confusion could happen.

`!!this.dateFilterState?.range`  returns a boolean based on whether `this.dateFilterState?.range` is truthy or falsy.
for example:
```
this.dateFilterState?.range = {start: '...', end: '...'}   ==>  (!!this.dateFilterState?.range) equals true
```

```
this.dateFilterState?.range = null  ==>  (!!this.dateFilterState?.range) equals false
```

for reference: https://developer.mozilla.org/en-US/docs/Glossary/Truthy

Reply:
Yes, but why was it working before without that :P

Reply:
It's not something that would break anything. 

It just that we were relying on truthy/falsy values  and now we are relying on true/false values just for more clarity but the logic is exactly the same it's totally equivalent.